tutorials/feast/README.md

```diff
@@ -0,0 +1,35 @@
+# Question Answering
+
+This tutorial explores how to leverage Feast for generating training data and enhancing online model inference. In this
+use-case, your goal is to train a ride-sharing driver satisfaction prediction model.
```

Comment:
"This tutorial explores how to leverage Feature Store, Feast, to reuse/share training features and provide consistency/efficiency in near real-time model inference. We will demonstrate the use-case through training a ride-sharing driver satisfaction prediction model."